src/Nethermind/Nethermind.Runner/Ethereum/Steps/RegisterRpcModules.cs

```diff
@@ -194,10 +164,18 @@ public virtual Task Execute(CancellationToken cancellationToken)
                 _api.EngineSignerStore,
                 _api.KeyStore,
                 _api.LogManager);
-
             _api.RpcModuleProvider.Register(new SingletonModulePool<IParityModule>(parityModule, true));
+
+            Web3Module web3Module = new Web3Module(_api.LogManager);
+            _api.RpcModuleProvider.Register(new SingletonModulePool<IWeb3Module>(web3Module, true));
+            
+            foreach (INethermindPlugin plugin in _api.Plugins)
+            {
+                await plugin.InitRpcModules();
```

Comment:
probably we need dependency on InitializePlugins step

Reply:
yeah, good point, thanks

Reply:
fixed